deploy/docker-compose.yml

```diff
@@ -0,0 +1,174 @@
+version: "3.9"
+volumes:
+  serai-base:
+  serai-alice:
+  serai-bob:
+  serai-charlie:
+  serai-dave:
+  serai-eve:
+  serai-ferdie:
+
+    # ------------------------------------------------------------
```

Comment:
Random ident?

Reply:
This is correct for yml. Only at the top level can a comment be left without indents.

Reply:
Thanks for clarifying.